### DISCUSSION TITLE
Suggestion: Work mode / context

### DISCUSSION THREAD

**Discussion by NatoNathan** [community/community#29266] [Other Feature Feedback, Questions, & Ideas]
We all love using GitHub for work, personal and open source projects. However it's important to keep certain elements of our lives separate, to maintain a healthy work-life balance.

It's recommended to only have a single GitHub account, and I agree with this recommendation. However it does have a few drawbacks such as.
- your notifications being filled with notifications for both work and personal projects
- lists of your repositories being overwhelmed with work, personal and open source repos
- long lists of organisations you have access to
- and much more

It would be nice to be able tailor your GitHub experience depending on if you're at work, working on personal stuff or helping open source projects. this mode / context could also change automatically based on time or even with SSO detection.

The things you can tailor would be organisation and repo visibility (as hiding personal orgs and repos at work and vice versa), Setting a status and even GitHub's theme. A cool feature could be viewing "hidden" repos and orgs in public view.

The goal would be to allow GitHub users experience to be as uncluttered as possible and keeping a clear separation between work and personal life.

**Comment by airtower-luna** [comment]
This makes me think of an option to hide/delay notifications from certain sources (repositories/orgs, maybe even categories within them) based on time. E.g. if a repository is marked as "work" notifications are hidden outside working hours.

Of course that wouldn't have to be limited to work/personal, a generic approach could be categories and hide/show settings for each category.
